<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BiswaranjanD-PC" name="Command line test 268be4de-b3e3-4798-9b60-eca7d852d4cc" tests="11" failures="5" timestamp="16 Aug 2017 05:23:11 GMT" time="1291.011" errors="0">
  <testcase name="payMarchant" time="80.173" classname="com.loylty.mobileautomation.testcases.Bom_Test"/>
  <testcase name="recharge" time="101.989" classname="com.loylty.mobileautomation.testcases.Bom_Test">
    <failure type="java.lang.AssertionError" message="BOM recharge unsucessfull expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: BOM recharge unsucessfull expected [true] but found [false]
	at com.loylty.mobileautomation.testcases.Bom_Test.recharge(Bom_Test.java:106)
	at com.loylty.automationframework.TestRunner.main(TestRunner.java:116)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
	at java.lang.Thread.run(Thread.java:745)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- recharge -->
  <testcase name="recharge_Guest" time="80.687" classname="com.loylty.mobileautomation.testcases.Bom_Test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.loylty.mobileautomation.testcases.Bom_Test.recharge_Guest(Bom_Test.java:127)
	at com.loylty.automationframework.TestRunner.main(TestRunner.java:116)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
	at java.lang.Thread.run(Thread.java:745)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- recharge_Guest -->
  <testcase name="verify_humberger" time="48.091" classname="com.loylty.mobileautomation.testcases.Bom_Test"/>
  <testcase name="payMarchant_Tmb" time="107.759" classname="com.loylty.mobileautomation.testcases.Tmb_Test"/>
  <testcase name="recharge_tmb" time="144.761" classname="com.loylty.mobileautomation.testcases.Tmb_Test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.loylty.mobileautomation.testcases.Tmb_Test.recharge_tmb(Tmb_Test.java:84)
	at com.loylty.automationframework.TestRunner.main(TestRunner.java:116)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
	at java.lang.Thread.run(Thread.java:745)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- recharge_tmb -->
  <testcase name="recharge_tmb_Guest" time="134.705" classname="com.loylty.mobileautomation.testcases.Tmb_Test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.loylty.mobileautomation.testcases.Tmb_Test.recharge_tmb_Guest(Tmb_Test.java:99)
	at com.loylty.automationframework.TestRunner.main(TestRunner.java:116)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
	at java.lang.Thread.run(Thread.java:745)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- recharge_tmb_Guest -->
  <testcase name="verify_humberger" time="47.76" classname="com.loylty.mobileautomation.testcases.Tmb_Test"/>
  <testcase name="registration_SBI" time="418.158" classname="com.loylty.mobileautomation.testcases.SBI_Test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.loylty.mobileautomation.testcases.SBI_Test.registration_SBI(SBI_Test.java:66)
	at com.loylty.automationframework.TestRunner.main(TestRunner.java:116)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
	at java.lang.Thread.run(Thread.java:745)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- registration_SBI -->
  <testcase name="verify_humberger_SBI" time="17.155" classname="com.loylty.mobileautomation.testcases.SBI_Test"/>
  <testcase name="verify_humberger_Canara" time="79.728" classname="com.loylty.mobileautomation.testcases.CanaraTest"/>
</testsuite> <!-- Command line test 268be4de-b3e3-4798-9b60-eca7d852d4cc -->
